### ISSUE TITLE
Villagers and My children hate me

### ISSUE THREAD

**Issue by ChickenYouAte** [WildBamaBoy/minecraft-comes-alive#1297]
**Describe the bug**
i literally can not talk to my children without them hating me, i click chat one time and its a positive message but if i do it any other time after that, i will continually get bad hearts infact the only person i can chat with without losing hearts at all is my wife who is at 1100 hearts rn, i also turned off diminishing returns and they still hate me,
**Steps to Reproduce**
Please include steps we can take to reproduce the bug:
1. Launch Minecraft
2. Get a family
3. talk to your children

**Minecraft Information (please complete all questions entirely):**
 - Minecraft Version: 1.12.2
 - Forge Version: 14.23.5.2838
 - MCA Version: 6.0.1
 - Launcher: Twitch

**Modpack Information**
Complete this section if you are using a modpack.
 - Modpack Name: SevTech Ages
 - Modpack Version: 3.1.2

**Comment by TsundereMatrix** [comment]
Have you tried giving your children gifts like sugar, diamonds, gold, etc?

**Comment by ChickenYouAte** [comment]
Yes, ive got into creative mode to experiment, i given them cookies,cake,gold blocks, gold ingots, diamond blocks, diamonds, swords, axes, and they hate all of it, wont even accept it once

**Comment by TsundereMatrix** [comment]
Cake?
